snap/hooks/install-device

```diff
@@ -9,3 +9,6 @@ net.ipv4.neigh.default.gc_thresh2=15360
 net.ipv4.neigh.default.gc_thresh3=16384
 net.netfilter.nf_conntrack_max=1048576
 EOF
+
+# NOTE: keep this in sync with the value written to 50-aws-gadget.conf
+echo 1048576 > /proc/sys/net/netfilter/nf_conntrack_max
```

Comment:
But this would be temporary for the install boot, then you reboot into normal run mode. Note also that when eventually we start to use preseeding, you do not want an install-device hook as it prevents installation from the initramfs. This should be in the install and post-refresh hooks (the latter in case the setting changes for a new gadget). After first boot in run mode, sysctl configuration should be enough.

Reply:
done (only `install` hook for now)